rewrite-java/src/main/java/org/openrewrite/java/ReplaceThreadRunWithThreadStart.java

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright 2022 the original author or authors.
+ * <p>
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * <p>
+ * https://www.apache.org/licenses/LICENSE-2.0
+ * <p>
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.openrewrite.java;
+
+import java.util.Collections;
+import java.util.Set;
+
+public class ReplaceThreadRunWithThreadStart extends ChangeMethodInvocation {
```

Comment:
It could be added as a declarative recipe, since it only invokes another recipe with parameters.
But I'm not sure it's possible to define tags in declarative recipe.

Anyway, IMO, delegating to the `ChangeMethodInvocation` recipe would be better than extending from it.